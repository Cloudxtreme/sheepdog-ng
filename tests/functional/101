#!/bin/bash

# Test sheep recovery logic when node of number of copies down

. ./common

for i in `seq 0 3`; do
    _start_sheep $i
done

_wait_for_sheep 4

_cluster_format -c 2

$DOG vdi create test 100M -P

_kill_sheep 1 &
_kill_sheep 2 &
_wait_for_sheep_recovery 0

_start_sheep 2 &
_wait_for_sheep_recovery 0
$DOG vdi check test
