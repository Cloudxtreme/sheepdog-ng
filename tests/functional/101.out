QA output created by 101
using backend plain store
finish check&repair test
